.travis.yml

```diff
@@ -10,14 +10,16 @@ cache:
   yarn: true
   directories:
     - "${HOME}/virtualenv/python$(python -c 'import platform; print(platform.python_version())')"
-    - '$NODE_DIR'
+    - '$VOLTA_HOME'
     - node_modules
     - '${HOME}/google-cloud-sdk'
 
 addons:
   apt:
     update: true
     packages:
+      # Volta links against openssl 1.0, which xenial's libssl-dev provides.
```

Comment:
I mean, there's also the openssl 1.1 version. This comment is helpful but not very clear about how/if/when we can remove installing this dependency.

Reply:
Sure, I can update it to be more helpful.

Also, I have a PR to use rustls so that their shipped binaries don't link against openssl for TLS: https://github.com/volta-cli/volta/pull/754